xml/http_servers.xml

```diff
@@ -1766,9 +1766,10 @@
       <param pos="1" name="service.version"/>
    </fingerprint>
 
-   <fingerprint pattern="^PHP/(\S+)$">
-      <example>PHP/4.4.2-1build1</example>
-      <example>PHP/5.1.2</example>
+   <fingerprint pattern="^PHP/(\S+)(?:\s.+)?$">
+      <example service.component.version="4.4.2-1build1">PHP/4.4.2-1build1</example>
+      <example service.component.version="5.1.2">PHP/5.1.2</example>
+      <example service.component.version="5.3.14">PHP/5.3.14 ZendServer/5.0</example>
```

Comment:
Yeah, I don't believe there is a good way to be doing this today, in Recog or the projects that utilize Recog.

For now, I'd suggest still breaking these into two fingerprints, but in the one that will match ZendZerver, don't assert ZendServer and simply leave a TODO comment.

This will help ensure that if other things come along which are not ZendServer they will not match and we'll stand a chance of identifying these mismatches and handling them as needed rather than letting them get absorbed in a catch-all.